composer.json

```diff
@@ -16,26 +16,49 @@
         "bin-dir": "bin"
     },
     "require": {
-        "neos/demo": "3.0.x-dev",
-        "typo3/neos-seo": "~1.0",
-        "typo3/setup": "~2.0",
-        "flowpack/neos-frontendlogin": "~2.0",
-        "neos/redirecthandler-neosadapter": "~1.0",
-        "neos/redirecthandler-databasestorage": "~1.0",
         "neos/neos": "3.0.x-dev",
+        "neos/demo": "3.0.x-dev",
+
+        "neos/seo": "2.0.x-dev",
+        "neos/setup": "4.0.x-dev",
+        "flowpack/neos-frontendlogin": "3.0.x-dev",
+        "neos/redirecthandler-neosadapter": "@dev",
+        "neos/redirecthandler-databasestorage": "@dev",
         "neos/nodetypes": "3.0.x-dev",
         "neos/site-kickstarter": "3.0.x-dev",
         "neos/content-repository": "3.0.x-dev",
         "neos/fusion": "3.0.x-dev",
         "neos/media": "3.0.x-dev",
-        "neos/diff": "3.0.x-dev"
+        "neos/diff": "3.0.x-dev",
+
+        "neos/flow": "4.0.x-dev",
+        "neos/fluid-adaptor": "4.0.x-dev",
+        "neos/eel": "4.0.x-dev",
+        "neos/cache": "4.0.x-dev",
+        "neos/twitter-bootstrap": "3.0.x-dev",
+        "neos/party": "4.0.x-dev",
+        "neos/form": "3.0.x-dev",
+        "neos/kickstarter": "4.0.x-dev",
+        "neos/media-browser": "3.0.x-dev",
+        "neos/imagine": "3.0.x-dev",
+        "neos/redirecthandler": "@dev",
```

Comment:
@Sebobo: Please try again. The neos/redirecthandler currently needs the @dev due to the version constraints of the redirecthandler packages. The others are (branched and) fixed.